hw/ip_templates/racl_ctrl/rtl/racl_ctrl.sv.tpl

```diff
@@ -2,27 +2,34 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
-module ${module_instance_name} import ${module_instance_name}_reg_pkg::*; #(
-  parameter logic [NumAlerts-1:0] AlertAsyncOn      = {NumAlerts{1'b1}},
-  parameter int unsigned          NumSubscribingIps = 1,
-  parameter bit                   RaclErrorRsp      = 1'b1
+module ${module_instance_name} 
+  import ${module_instance_name}_reg_pkg::*;
+  import top_racl_pkg::*;
+#(
+  parameter logic [NumAlerts-1:0] AlertAsyncOn              = {NumAlerts{1'b1}},
+  parameter int unsigned          NumSubscribingIps         = 1,
+  parameter int unsigned          NumExternalSubscribingIps = 1,
+  parameter bit                   RaclErrorRsp              = 1'b1
 ) (
-  input  logic                                                 clk_i,
-  input  logic                                                 rst_ni,
+  input  logic                                                         clk_i,
+  input  logic                                                         rst_ni,
 % if enable_shadow_reg:
-  input logic                                                  rst_shadowed_ni,
+  input logic                                                          rst_shadowed_ni,
 % endif
   // Bus Interface (device)
-  input  tlul_pkg::tl_h2d_t                                    tl_i,
-  output tlul_pkg::tl_d2h_t                                    tl_o,
+  input  tlul_pkg::tl_h2d_t                                            tl_i,
+  output tlul_pkg::tl_d2h_t                                            tl_o,
   // Alerts
-  input  prim_alert_pkg::alert_rx_t [NumAlerts-1:0]            alert_rx_i,
-  output prim_alert_pkg::alert_tx_t [NumAlerts-1:0]            alert_tx_o,
+  input  prim_alert_pkg::alert_rx_t [NumAlerts-1:0]                    alert_rx_i,
+  output prim_alert_pkg::alert_tx_t [NumAlerts-1:0]                    alert_tx_o,
   // Output policy vector for distribution
-  output top_racl_pkg::racl_policy_vec_t                       racl_policies_o,
+  output top_racl_pkg::racl_policy_vec_t                               racl_policies_o,
   // RACL violation information.
-  input logic            [NumSubscribingIps-1:0]               racl_error_i,
-  input top_racl_pkg::racl_error_log_t [NumSubscribingIps-1:0] racl_error_log_i
+  input logic                          [NumSubscribingIps-1:0]         racl_error_i,
+  input top_racl_pkg::racl_error_log_t [NumSubscribingIps-1:0]         racl_error_log_i,
```

Comment:
Nit: you can drop `top_racl_pkg` here as it's important above already.

Reply:
Unfortunately not. For out-of-tree connections, we use a custom stiching-tool that relies on the full-scope package type (and we'll use that IP externally). I removed the import at the very top and added the import within the module body.